libs/domains/services/feature/src/lib/service-details/service-details.tsx

```diff
@@ -0,0 +1,221 @@
+      ({ maximum_cpu, maximum_memory, max_duration_seconds, max_nb_restart, schedule, port }) => (
+        <>
+          <ResourceUnit value={formatCronExpression(schedule?.cronjob?.scheduled_at)} description="Scheduling" />
+          <ResourceUnit value={max_nb_restart} description="Restart (max)" />
+          <ResourceUnit value={`${max_duration_seconds} s`} description="Duration (max)" />
+          <ResourceUnit value={maximum_cpu} description="VCPU (max)" />
+          <ResourceUnit value={formatBytes(maximum_memory)} description="Memory (max)" />
+          <ResourceUnit value={port} description="Port" />
+        </>
+      )
+    )
+    .otherwise(() => null)
+
+  const sectionClassName = 'text-neutral-350 gap-4 pl-8 pr-5'
+
+  return (
+    <div className={twMerge('flex flex-col shrink-0 gap-5 py-8 text-sm', className)} {...props}>
+      <Section className={sectionClassName}>
+        {/* XXX: Should be Heading, typography & design wanted */}
+        <span className="text-neutral-400 font-medium">About</span>
+        <p>{service.description || 'No description provided yet'}</p>
+        <Dl>
+          <Dt>Created:</Dt>
+          <Dd>{dateFullFormat(service.created_at)}</Dd>
+          {service.updated_at && (
+            <>
+              <Dt>Last edit:</Dt>
+              <Dd>
+                <Tooltip content={dateFullFormat(service.updated_at)}>
+                  <span>{timeAgo(new Date(service.updated_at))}</span>
+                </Tooltip>
+              </Dd>
+            </>
+          )}
+        </Dl>
+      </Section>
+      <hr />
+      <Section className={sectionClassName}>
+        {/* XXX: Should be Heading, typography & design wanted */}
+        <span className="text-neutral-400 font-medium">Source</span>
+        {match(service)
+          .with({ serviceType: ServiceTypeEnum.APPLICATION }, { serviceType: ServiceTypeEnum.JOB }, (service) => {
+            const gitRepository = match(service)
+              .with({ serviceType: ServiceTypeEnum.APPLICATION }, ({ git_repository }) => git_repository)
+              .with({ serviceType: ServiceTypeEnum.JOB }, ({ source }) => source?.docker?.git_repository)
+              .exhaustive()
+
+            return (
+              gitRepository && (
+                <Dl>
+                  {gitRepository.url && gitRepository.name && (
+                    <>
+                      <Dt>Repository:</Dt>
+                      <Dd>
+                        <a href={gitRepository.url} target="_blank" rel="noopener noreferrer">
+                          <Badge variant="surface" size="xs" className="gap-2">
+                            <Icon
+                              name={gitRepository.url.includes('//github') ? IconEnum.GITHUB : IconEnum.GITLAB}
+                              height={14}
+                              width={14}
+                            />
+                            <Truncate text={gitRepository.name} truncateLimit={20} />
```

Comment:
The repository "tags" overflows a little onto the right padding

<img width="372" alt="Capture d’écran 2023-10-02 à 11 38 33" src="https://github.com/Qovery/console/assets/533928/a00b117a-31c3-4004-a14c-c124d02a5f42">

Reply:
I think you haven't test the latest version.